core/src/main/scala/com/spotify/featran/FeatureBuilder.scala

```diff
@@ -180,60 +179,9 @@ case class NamedSparseArray[@specialized(Float, Double) T](
 }
 
 object FeatureBuilder {
-  private final case class ArrayFB[T: ClassTag: FloatingPoint](
-    private var underlying: Array[T] = null
-  ) extends FeatureBuilder[Array[T]] {
-    private var offset: Int = 0
-
-    override def init(dimension: Int): Unit = underlying = new Array[T](dimension)
-
-    override def add(name: String, value: Double): Unit = {
-      underlying(offset) = FloatingPoint[T].fromDouble(value)
-      offset += 1
-    }
-
-    override def skip(): Unit = offset += 1
-
-    override def skip(n: Int): Unit = offset += n
-
-    override def result: Array[T] = {
-      require(offset == underlying.length)
-      offset = 0
-      underlying.clone()
-    }
-
-    override def newBuilder: FeatureBuilder[Array[T]] = ArrayFB()
-  }
-
-  implicit def arrayFB[T: ClassTag: FloatingPoint]: FeatureBuilder[Array[T]] = ArrayFB()
-
-  // Workaround for CanBuildFrom not serializable
-  trait CanBuild[T, M] extends Serializable {
-    def apply(): mutable.Builder[T, M]
-  }
-
-  private def newCB[T, M](f: () => mutable.Builder[T, M]) = new CanBuild[T, M] {
-    override def apply(): mutable.Builder[T, M] = f()
-  }
-
-  // Collection types in _root_.scala.*
-  //scalastyle:off public.methods.have.type
-  implicit def traversableCB[T] = newCB(() => Traversable.newBuilder[T])
```

Comment:
we could keep these and ref the new ones under `CanBuild` implicit scope. However we can always just remove them and bum,p version to `0.5.0`

Reply:
MiMa will be mainly complain about this ... if we bump version I don't care that much about bin compat

Reply:
Yeah plus people really shouldn't be using `Traversable` directly. This was supposed to be a catch all for `predef` collection types.